gcsio/src/main/java/com/google/cloud/hadoop/gcsio/testing/InMemoryGoogleCloudStorage.java

```diff
@@ -51,6 +51,11 @@
  */
 public class InMemoryGoogleCloudStorage implements GoogleCloudStorage {
 
+  private static final CreateObjectOptions EMPTY_OBJECT_CREATE_OPTIONS =
```

Comment:
I see you created this several times, should it just go to CreateObjectOptions.java as a public static variable then?

Reply:
This is a test class - it has a lot of duplicate code, but I guess this is WAI, because you do not want silently change tests, when you changing prod code, otherwise tests are less reliable.